students/chylak_malgorzata/lesson_03_functions/refactor_calculator.py

```diff
@@ -0,0 +1,120 @@
+def add():
+    """Add function docstring"""
+    print("ADDING")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 + add_var_2)
+
+
+def subtract():
+    """Subtract function docstring"""
+    print("SUBTRACT")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
```

Comment:
In each function, you repeat lines from 15 to 18 so, they could be moved to separate function.